.github/workflows/sync_typeshed.yml

```
@@ -0,0 +1,31 @@
+name: Sync typeshed
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 1,15 * *"
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  sync_typeshed:
+    name: Sync typeshed
+    if: github.repository == 'python/typeshed'
```

```suggestion
    if: github.repository == 'python/mypy'
```
More copy-paste problems? :)